pkg/log/log.go

```diff
@@ -104,11 +104,14 @@ var (
 
 // InitLogger initializes DM's and also the TiDB library's loggers.
 func InitLogger(cfg *Config) error {
-	err := logutil.InitLogger(&logutil.LogConfig{Config: pclog.Config{Level: cfg.Level}})
-	if err != nil {
-		return terror.ErrInitLoggerFail.Delegate(err)
+	// init and set tidb slow query logger to stdout if log level is debug
+	if cfg.Level == "debug" {
+		confG := &pclog.Config{Level: cfg.Level}
+		slowQueryLogger, _, _ := pclog.InitLogger(confG)
```

Comment:
inside `InitLogger` there's `registerOnce.Do`, so at first glance it's not expected to call twice

Reply:
it's ok to call `InitLogger` muti times, add a test `TestInitSlowQueryLogger` for this
 https://github.com/pingcap/dm/pull/1910/commits/532e0ffb0037bd5924012571aa598c95c7092bac

Reply:
use `zap.NewExample` to init a new debug-level logger, and this does not affect the configuration of the encoder inside the zap  https://github.com/pingcap/dm/pull/1910/commits/e7eab2087aa70dad9e2a0d4dbd096d84f14354dd